Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -762,7 +690,7 @@ public function hasMethod(string $className, string $methodName): bool
      * Returns all tags and their values the specified method is tagged with
      *
      * @throws ReflectionException
-     * @api
+     * @deprecated since 8.4
      */
     public function getMethodTagsValues(string $className, string $methodName): array
```

Comment:
i suppose this is getting deprecated because its unused system wide?

Reply:
Yep, correct, there is a test for it, but otherwise it's completely unused.